File: 7-kyu/fizz-buzz/python/solution.py
def fizzbuzz_func(n):
    ret = ''
    if n % 3 == 0: ret += 'Fizz'
    if n % 5 == 0: ret += 'Buzz'
    if not ret: ret = n
    return ret

def fizzbuzz(n):
    return list(map(fizzbuzz_func, [i for i in range(1, n + 1)]))

File: 7-kyu/fizz-buzz/javascript/solution.js
const fizzbuzz = n => {
  let ret = [];
  for (let i = 1; i <= n; i++) {
    if (i % 15 == 0) {
      ret.push("FizzBuzz");
    } else if (i % 5 == 0) {
      ret.push("Buzz");
    } else if (i % 3 == 0) {
      ret.push("Fizz");
    } else {
      ret.push(i);
    }
  }
  return ret;
};
